### PR TITLE
Removed Job Poster Creation from Managers

### LINKED ISSUE #817: Manager Ready: Removing Job Poster Creation from Managers
# Description
While we wait on the new job poster builder (currently in design and testing), we're going to remove the ability to create job posters from managers. In exchange, we'll allow admins to create these by providing a UI where the admin can select a manager from a list, and click a button to generate a job poster attached to that manager. This eliminates the immediate need for validation on this form (we won't be using it once the builder is in place), and it eliminates the worry that managers will arbitrarily create posters and run into a validation bug.

## Criteria
- [x] Remove "Create a Job Poster" button from manager UI.
- [x] Add an Admin UI that allows job poster creation:
  - [x] Admins should be able to select a manager from a dropdown.
  - [x] Admins should be able to then click a button to create a job poster attached to that manager, as if they had created it themselves.
- [x] This task requires a unit test.

## Wireframes
image

### PR DESCRIPTION
**Pull request by tristan-orourke**
Added a Managers page to the Admin portal, where admins can create a job poster for any manager. Removed the Create Job Poster menu item from the manager portal, and update permissions so Managers can update, but not create, job posters.
Resolves #817

### CODE REVIEW

File: tests/Feature/JobControllerTest.php [hunk 1]
```diff
@@ -173,42 +174,59 @@ public function testSubmitForReview() : void
...
      *
      * @return void
      */
-    public function testManagerCreate() : void
+    public function testCreateAsManager() : void
     {
-        $newJob = $this->generateEditJobFormData();
-
-        $dbValues = array_slice($newJob, 0, 8);
+        $admin = factory(User::class)->state('admin')->create();
```
Comment (Khristophor) [inline tests/Feature/JobControllerTest.php]: It looks like it's `states()` in the documentation.

```php
$users = factory(App\User::class, 5)->states('delinquent')->make();
```

File: tests/Feature/JobControllerTest.php [hunk 2]
```diff
@@ -291,16 +312,20 @@ public function testSavedJobHasCorrectTimes() : void
...
+        $jobEdit['close_date'] = $closeDate;
 
         //Expected db values
-        $dbValues = array_slice($newJob, 0, 8);
+        $dbValues = array_slice($jobEdit, 0, 8);
 
         $response = $this->followingRedirects()
-            ->actingAs($this->manager->user)
-            ->post('manager/jobs/', $newJob);
+            ->actingAs($this->manager)
```
Comment (Khristophor) [inline tests/Feature/JobControllerTest.php]: Have to pass `$this->manager->user` to `actingAs()` here.

File: tests/Feature/JobControllerTest.php (hunk 1 above)
Comment (tristan-orourke) [reply tests/Feature/JobControllerTest.php]: fixed

File: tests/Feature/JobControllerTest.php (hunk 2 above)
Comment (tristan-orourke) [reply tests/Feature/JobControllerTest.php]: Thanks, fixed

**Pull request merged**